### ISSUE TITLE
Issue running a command on multiple hosts in curses mode

### ISSUE THREAD

**Issue by StuartUSA** [zlaxy/sshch#12]
Firstly, thanks for a neat tool. 

Secondly, when in curses mode I select multiple aliases then press 'c' I enter a command that I expect to run on the hosts that I had selected, however the command is executed on the localhost.

Not sure if it the correct fix, but I changed line 266 from:
ConnectAlias(alias, command)
to:
ConnectAlias(alias.split()[0], command)

The variable alias at this point contains the alias and group, this fails to resolve.

Thanks
Stuart

**Comment by zlaxy** [comment]
@StuartUSA hello, the current repository is here: https://gitlab.com/zlax/sshch
I don't use Microsoft products, so i stopped using github.com. I also don't recommend you to use Microsoft products.

You can check this issue on the latest version of sshch on gitlab. In recent versions it is now possible to execute commands in parallel mode.

```
'c'/'F3' - command execution for alias (group/aliases - in turn)

't'/'F11' - parallel command execution for aliases and groups (ssh key authentication or set password required)
```

You can also check this functionality for your tasks and if there are any questions i ask you to open your new issue on https://gitlab.com/zlax/sshch

Also i ask you to close this issue here, on github.

**Comment by StuartUSA** [comment]
Thank you, I have changed repositories.
